### ISSUE TITLE
Feature request: Selectable time mode difficulty

### ISSUE THREAD

**Issue by sm4rk0** [LonamiWebs/Klooni1010#18]
Something like "Easy", "Medium" and "Hard".
I know it's [already asked for](https://github.com/LonamiWebs/Klooni1010/issues/6#issuecomment-280689156), but just to keep it as a separate issue.
To other users with the same wish, please use the "add your reaction" button in top-right corner of this comment, instead of commenting with "me, too", or "+1".

**Comment by XyLoNaMiyX** [comment]
How should this be presented to the user?

**Comment by sm4rk0** [comment]
Currently, there's one "play" button (1) occupying whole width, and three buttons (2 - "star", 3 - "clock" and 4 - "palette") in a row below it:
111
234

One possible solution that keeps the UI/UX clean and one-click:
111
abc
2|4
Where "a", "b" and "c" are "Easy", "Medium", "Hard" with a clock in different colors.

Or maybe this layout:
111
aaa
bbb
ccc
2|4

Better idea anyone?

**Comment by RustanHakansson** [comment]
Showing the game mode on the start screen would be useful. Right now there is no way to know what game mode you are going to play before starting. Clicking on the game mode to see a dropdown and be able to switch there immediately would be useful, especially if additional game modes are added later #70 . Having multiple options for time mode would be an easy way to do things:

Game mode:
Classic
Time - easy
Time - medium
Time - hard

**Comment by Lonami** [comment]
> Right now there is no way to know what game mode you are going to play before starting

Yes there is, I explained this in my comment at https://github.com/LonamiWebs/Klooni1010/issues/70#issuecomment-685069733. Green button normal mode. Blue for timer mode.

**Comment by RustanHakansson** [comment]
> > Right now there is no way to know what game mode you are going to play before starting
> 
> Yes there is, I explained this in my comment at [#70 (comment)](https://github.com/LonamiWebs/Klooni1010/issues/70#issuecomment-685069733). Green button normal mode. Blue for timer mode.

My bad, I did not at all understand that icon, I did not even try to click it. I must have accidentally clicked it once.

**Comment by Lonami** [comment]
Gonna close this issue as frankly I remember nothing about the project so it's unlikely I'll change it further.
